components/CollectiveTypePicker.js

```diff
@@ -46,9 +41,24 @@ const CollectiveTypePicker = ({ types, onChange }) => {
           borderRadius={8}
```

Comment:
To match design, we need to update the padding y of this button to `28px`

![image](https://user-images.githubusercontent.com/1556356/84359515-0c381680-abc9-11ea-88fa-ec096818ac2d.png)